packages/react-adminer/src/components/List.tsx

```diff
@@ -139,13 +145,24 @@ const ListChild: React.FC<Props> = ({ entityConfig, entityName, filter = true })
 		}
 	};
 
+	const getColumnTitle = (f: TableField): any => {
```

Comment:
1. tohle se bude používat víc -> do utilitních funkcí a ať to vrací jen ten text
2. Proč to vrací any?

--